src/components/input-panel/index.js

```diff
@@ -27,9 +27,13 @@ class InputPanel extends React.Component {
 
     let outerComponent;
     if (innerPanes.length > 1) {
-      outerComponent = React.createElement(SplitPane, {
-        split: 'horizontal'
-      }, innerPanes);
+      if (this.props.mode === MODES.VegaLite) {
+        outerComponent = React.createElement('div', {style: {width: '100%'}}, innerPanes);
+      } else {
+        outerComponent = React.createElement(SplitPane, {
+          split: 'horizontal'
+        }, innerPanes);
+      }
```

Comment:
Would be awesome if we could get this working with the `SplitPane` and not have to special case when mode is VegaLite